### PR TITLE
NIFI-6179: minor changes to DruidTranquilityController

### PR DESCRIPTION
**Pull request by ekovacs**
- hooking in property INDEX_RETRY_PERIOD to list of properties
- fixing property description

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by ekovacs** [comment]
@mattyb149 @arpadboda  can you take a look?

File: nifi-nar-bundles/nifi-druid-bundle/nifi-druid-controller-service/src/main/java/org/apache/nifi/controller/druid/DruidTranquilityController.java [hunk 1]
```diff
@@ -259,7 +259,7 @@
     public static final PropertyDescriptor INDEX_RETRY_PERIOD = new PropertyDescriptor.Builder()
             .name("druid-cs-index-retry-period")
             .displayName("Index Retry Period")
-            .description("Grace period to allow late arriving events for real time ingest.")
+            .description("Time period, untill a transiently failing indexing service overlord call is retried, before giving up.")
```
Comment (arpadboda) [inline nifi-nar-bundles/nifi-druid-bundle/nifi-druid-controller-service/src/main/java/org/apache/nifi/controller/druid/DruidTranquilityController.java]: Typo: until

**Comment by arpadboda** [comment]
https://nifi.apache.org/docs/nifi-docs/components/org.apache.nifi/nifi-druid-nar/1.9.1/org.apache.nifi.controller.druid.DruidTranquilityController/index.html

As far as I see this property is not documented, could you align documentation as well?

**Comment by ekovacs** [comment]
> https://nifi.apache.org/docs/nifi-docs/components/org.apache.nifi/nifi-druid-nar/1.9.1/org.apache.nifi.controller.druid.DruidTranquilityController/index.html
> 
> As far as I see this property is not documented, could you align documentation as well?

documentation will be generated from the property itself i believe. Here is a screenshot what i see in my local build:
Screen Shot 2019-04-03 at 16 25 21

File: nifi-nar-bundles/nifi-druid-bundle/nifi-druid-controller-service/src/main/java/org/apache/nifi/controller/druid/DruidTranquilityController.java (hunk 1 above)
Comment (ekovacs) [reply nifi-nar-bundles/nifi-druid-bundle/nifi-druid-controller-service/src/main/java/org/apache/nifi/controller/druid/DruidTranquilityController.java]: oops, added a typo :-/

**Comment by mattyb149** [comment]
+1 LGTM, ran unit tests, verified property is displayed to user (with associated documentation). Thanks for the fix! Merging to master

**Comment by ekovacs** [comment]
thank you @mattyb149 !

**Pull request closed without merging**